docs/design_proposals/sync-auto-support.md

```diff
@@ -0,0 +1,150 @@
+# Title
+
+* Author(s): Appu Goundan (@loosebazooka)
+* Design Shepherd: \<skaffold-core-team-member\>
```

Comment:
```suggestion
* Design Shepherd: Tejal Desai
```

Reply:
You can add me as a design shepherd as well!